FTBFS with vala-0.45.3

```
 * slick-greeter version - slick-greeter-1.2.6
 * Distribution - Fedora 31 (Rawhide)
 * vala-0.45.3
 * 64 bit
 ```

**Issue**

slick-greeter fails to compile with vala-0.45.3

```
BUILDSTDERR: make[2]: Leaving directory '/builddir/build/BUILD/slick-greeter/greeter-list.vala:167.5-167.22: error: Creation method of abstract class cannot be public.
BUILDSTDERR:     public GreeterList (Background bg, MenuBar mb)
BUILDSTDERR:     ^^^^^^^^^^^^^^^^^^
BUILDSTDERR: make[2]: *** [Makefile:1106: slick_greeter_vala.stamp] Error 1
BUILDSTDERR: make[2]: *** Waiting for unfinished jobs...
```


**Steps to reproduce**

Try to compile slick-greeter with vala-0.45.3


**Expected behaviour**

slick-greeter should compile.

**Other information**

--- a/src/greeter-list.vala
+++ b/src/greeter-list.vala
@@ -164,7 +164,7 @@ public abstract class GreeterList : FadableBox
         }
     }
 
-    public GreeterList (Background bg, MenuBar mb)
+    protected GreeterList (Background bg, MenuBar mb)
     {
         Object (background: bg, menubar: mb);
     }

